misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -103,12 +108,12 @@ public CoverageReport(Module module, Map<String, ModuleCoverage> moduleCoverageM
     /**
      * Generates the testerina coverage report.
      *
-     * @param jBallerinaBackend JBallerinaBackend
+     * @param jBallerinaBackend  JBallerinaBackend
      * @param includesInCoverage boolean
      * @throws IOException
      */
     public void generateReport(JBallerinaBackend jBallerinaBackend, String includesInCoverage,
-                               String reportFormat)
+                               String reportFormat, Map<String, Module> originalModules)
```

Comment:
Since we create a CoverageReport per module, I think we can only pass the corresponding original module without having to pass the entire map of original modules

Reply:
Yes. Im passing only the required module now instead of the map